tests/Fixtures/project/src/Aspect/InconsistentlyWeavingAspect.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Go\Tests\TestProject\Aspect;
+
+use Go\Aop\Aspect;
+use Go\Aop\Intercept\MethodInvocation;
+use Go\Tests\TestProject\Application\InconsistentlyWeavedClass;
+use Go\Lang\Annotation as Pointcut;
+
+class InconsistentlyWeavingAspect implements Aspect
+{
+    public function __construct(InconsistentlyWeavedClass $problematicClass)
+    {
+    }
+
+    /**
+     * Intercepts badlyWeaved()
+     *
+     * @param MethodInvocation $invocation
+     *
+     * @Pointcut\After("execution(public Go\Tests\TestProject\Application\InconsistentlyWeavedClass->badlyWeaved(*))")
```

Comment:
Make a note here as well, that we are trying to intercept method from the class that we want to use in the current aspect.